device/mellanox/x86_64-mlnx_msn2700-r0/platform_reboot

```diff
@@ -28,6 +28,17 @@ function SafePwrCycle() {
 
 ParseArguments "$@"
 
+# Reboot immediately if the kdump capture kernel is running
+VMCORE_FILE=/proc/vmcore
+if [ -e $VMCORE_FILE -a -s $VMCORE_FILE ]; then
```

Comment:
`-s` include `-e`, so you only need `-s`.

Reply:
fixed

Reply:
What is purpose of first `$VMCORE_FILE`? single `-s` is good enough?

Reply:
yes. fixed.